src/sciwyrm/main.py

```diff
@@ -44,12 +60,13 @@ def _inject_app_config(
         raise RequestValidationError(errors) from None
 
 
-@app.post("/notebook", response_model=dict, response_description="Rendered notebook")
-async def format_notebook(
+@app.post("/notebook", response_model=dict, response_description="Rendered notebook with json input")
```

Comment:
```suggestion
@app.post("/notebook", response_model=dict, response_description="Rendered notebook from json input")
```
It doesn't return the json intput.